src/ipc/ipc3/dai.c

```diff
@@ -156,6 +160,8 @@ int ipc_dai_data_config(struct dai_data *dd, struct comp_dev *dev)
 		 */
 		dd->stream_id = config->alh.stream_id;
 		break;
+	case SOF_DAI_IMX_MICFIL:
+		COMPILER_FALLTHROUGH;
```

Comment:
usually we don't use `COMPILER_FALLTHROUGH` if there's no code between `case` lines - see lines 173-174 below

Reply:
I don't remember the reason we had these, but I just followed the existing code. I will remove it and have proper code.